### DISCUSSION TITLE
Color schemes

### DISCUSSION THREAD

**Discussion by airald** [devemux86/cruiser#313] [Show and tell]
Just want to show the different colour schemes I use depending on the profile.

Day motorcycle profile:
![Screenshot_2023-09-07-22-22-50-029_gr talent cruiser](https://github.com/devemux86/cruiser/assets/20151719/a1633d20-6abf-4958-962d-0ac91e49e729)
<img src="https://github.com/devemux86/cruiser/assets/20151719/c454e429-66eb-4dcf-8756-a303ff85c474" width="50%"/>

Night motorcycle profile:
![Screenshot_2023-09-08-13-22-18-255_gr talent cruiser~2](https://github.com/devemux86/cruiser/assets/20151719/cf9ac28e-5e7b-46a9-92c0-4368d6499eb5)
<img src="https://github.com/devemux86/cruiser/assets/20151719/f776e681-68bd-4988-bc25-55cc0f2809d1" width="50%"/>

For me is specially important the night profile, to achieve a good contrast with minimum screen light. 

In the car I use that one: 
<img src="https://github.com/devemux86/cruiser/assets/20151719/a3a5a7a8-33c9-4d16-bc8a-4739b546b345" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/20151719/6901e61f-166d-41db-85ad-d01d22f5885c" width="50%"/>

And this one for walking:
<img src="https://github.com/devemux86/cruiser/assets/20151719/bbb77b7a-888b-4c5e-ad0d-f46863c775e7" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/20151719/b9c28f00-c534-492a-9c77-ca461c22d90d" width="50%"/>

**Comment by devemux86** [comment]
@airald excellent!

I really like the very beautiful night profile! :+1: 
Is it with "Inverted" map color?

**Reply by airald** [reply]
Yes, inverted map. Normal one emits white light even at minimum.

**Comment by Ciutnik** [comment]
Now it happens to me rarely, but a few years ago I rode a motorcycle after dark every few days. But I would still love to have the app/map theme automatically change depending on conditions. Not only the time of day but also the tunnels are important. 

But I can imagine how extremely complicated it is! Each profile would have to have two color versions: "day" and "night". And each of these versions – the color of the map, applications, panels, fonts... 

A difficult thing, but probably to be done sooner or later 🙂

**Reply by devemux86** [reply]
@Ciutnik we have probably discussed this in the past, indeed it is a bit complicated and needs a good design.

But it is useful and it is already in my list to check how we can offer it.

**Reply by Ciutnik** [reply]
As for me, I would be willing to give up different colors for each profile separately, in exchange for only two variants common to all profiles: day and night. This would probably be much easier to implement, but I understand that since users got profile colors, it would be difficult to take it away from them now 🙂

**Reply by devemux86** [reply]
@Ciutnik see also the existing topic where we mention some of the things that could be different in day / night:
- https://github.com/devemux86/cruiser/discussions/126
